pkg/generators/informergen/informergen.go

```diff
@@ -213,6 +225,11 @@ func (g *Generator) GetGVKs(ctx *genall.GenerationContext) (map[types.Group]map[
 		}
 	}
 
+	for oldGroupName, newGroupName := range renamedGroups {
+		gvks[newGroupName] = gvks[oldGroupName]
+		delete(gvks, oldGroupName)
```

Comment:
so this generates the proper calls but messes up the import paths. Need to store the GoName in addition to the import name. Probably will make a wrapper around types.Group that includes this info.